pkg/eval/builtin_fn_styled.go

```diff
@@ -68,7 +68,12 @@ var errStyledSegmentArgType = errors.New("argument to styled-segment must be a s
 //     -   Any color from the xterm 256-color palette, as `colorX` (such as
 //    `color12`).
 //
-//     -   A 24-bit RGB color, as `#RRGGBB`, such as `#778899`.
+//     -   A 24-bit RGB color written as `#RRGGBB` such as `'#778899'`. Note
+//     that you need to quote such values since the `#` character introduces a
+//     comment if not in a quoted string. So `'#'778899`, `'#778899'`,
```

Comment:
I prefer `'#778899'` to `'#'778899` for the example.

Reply:
Done.